shell/src/main/java/alluxio/shell/command/CopyFromLocalCommand.java

```diff
@@ -230,7 +230,7 @@ private void copyPath(File src, AlluxioURI dstPath) throws IOException {
             buf.flip();
             os.write(buf.array(), 0, buf.limit());
           }
-        } catch (IOException e) {
+        } catch (IOException | RuntimeException e) {
```

Comment:
@aaudiber  Got it, but I new comer for this project, so do I open a new PR(close this one) to fix the code as your suggestion?